### PR TITLE
V2 compability tests

### PR DESCRIPTION
**Pull request by Olamyy**
*Description of changes:* Added tests for the ASTTransformer class.

*Testing done:* Yes

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ x ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ x ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to any/all clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ x ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ x ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/unit/v2/test_transformer.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        expected = """TensorFlow(entry_point='foo.py', framework_version='1.11.0')
+sagemaker.tensorflow.TensorFlow(framework_version='1.11.0')
+m = MXNet(framework_version='1.2.0')
+sagemaker.mxnet.MXNet(framework_version='1.2.0')\n"""
+
+        self.assertEqual(pasta.dump(rewrite), expected)
+
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (laurenyu) [inline tests/unit/v2/test_transformer.py]: we use pytest, so as long as a function name starts with "test", it is run as part of the test suite

**Comment by laurenyu** [comment]
for the build failure:

```
flake8 run-test: commands[0] | flake8
./tests/unit/v2/test_transformer.py:1:1: FI11 __future__ import "absolute_import" missing
```

also, can you run `tox -e black-format` and commit those changes?

File: tests/unit/v2/test_transformer.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
```
Comment (laurenyu) [inline tests/unit/v2/test_transformer.py:2]: can you move this file to https://github.com/aws/sagemaker-python-sdk/tree/zwei/tests/unit/sagemaker/cli/compatibility/v2?

File: tests/unit/v2/test_transformer.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
```
Comment (laurenyu) [inline tests/unit/v2/test_transformer.py:1]: please add this copyright license to the top:

```
# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License"). You
# may not use this file except in compliance with the License. A copy of
# the License is located at
#
#     http://aws.amazon.com/apache2.0/
#
# or in the "license" file accompanying this file. This file is
# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
# ANY KIND, either express or implied. See the License for the specific
# language governing permissions and limitations under the License.
```

File: tests/unit/v2/test_transformer.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+import ast
+from sagemaker.tools.compatibility.v2.ast_transformer import ASTTransformer
```
Comment (laurenyu) [inline tests/unit/v2/test_transformer.py:4]: the code has been moved a couple times, but now lives in `sagemaker.cli.compatibility.v2` (I think this should also fix the failing build)

**Comment by laurenyu** [comment]
also, can you rebase on top of the "zwei" branch?

**Comment by laurenyu** [comment]
@Olamyy have you had a chance to revisit this? It's pretty close to ready for merging, and I'd love to get it merged in soon

**Comment by ChuyangDeng** [comment]
Closing due to lack of acitivity.

**Pull request closed without merging**